tests/system/python/e2e/test_e2e_modbus_c_pi.py

```diff
@@ -64,7 +64,8 @@ def start_south_north(self, reset_and_start_foglamp, add_south, remove_directori
             remove_directories: Fixture that remove directories created during the tests
         """
         self.check_connect(modbus_host, modbus_port)
-        cfg = {"protocol": {"value": "TCP"}, "asset": {"value": ""}, "port": {"value": "502"},
+        cfg = {"protocol": {"value": "TCP"}, "asset": {"value": ""}, "address": {"value": modbus_host},
+               "port": {"value": "{}".format(modbus_port)},
```

Comment:
`"{}".format(modbus_port)` could just  be `str(modbus_port)`.  Though we are good as is!